src/util.js

```diff
@@ -1,8 +1,49 @@
 /*
  * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
  */
-import {asn1, oids, util} from 'node-forge';
-const {ByteBuffer} = util;
+
+/**
+ * Takes in a hexidecimal string and parses it 2 characters at a time
+ *  into a Uint8Array.
+ *
+ * @param {string} hexString - A hexidecimal string.
+ *
+ * @returns {Unit8Array} The resulting bytes.
+*/
+const fromHexString = hexString => {
+  const result = [];
+  // hexidecimal numbers should be 2 characters between 0-F
+  const hexCharLength = 2;
+  for(let i = 0; i < hexString.length; i += hexCharLength) {
+    // take the 2 characters and parse to a base 16 integer
+    result.push(parseInt(hexString.substr(i, hexCharLength), 16));
+  }
+  // return the resulting array as a Uint8Array interoperable in Node & Browser
+  return new Uint8Array(result);
+};
+
+// A Uint8Array with an private ed25519 DER prefix in it.
+export const DER_PRIVATE_KEY_PREFIX = fromHexString(
+  '302e020100300506032b657004220420');
+
+// A Uint8Array with a public ed25519 DEr prefix in it.
+export const DER_PUBLIC_KEY_PREFIX = fromHexString('302a300506032b6570032100');
```

Comment:
We don't need any of this in the browser, right? If we don't, then we also don't need `fromHexString` at all since `Buffer.from('<hex>', 'hex')` will do. We also don't need `combineUint8Arrays` as `Buffer.concat` will do.

Reply:
+1, all of the remaining contents of `util.js` should be moved to the node-only file.

Reply:
this library exports the methods:

```js
export {Ed25519VerificationKey2018} from './Ed25519VerificationKey2018.js';
export {base58Decode, privateKeyDerEncode, publicKeyDerEncode} from './util.js';
```

https://github.com/digitalbazaar/ed25519-verification-key-2018/blob/master/src/main.js

so basically in order to be cross-compatible we do have to do this, or make breaking changes to the API. I had intended this as a major release so that might not be a big deal. Let me know if that changes anything. I'll remove the DerEncode methods from the export and make them node specific.

Reply:
We should not be exporting those things from `util`. So +1 to making them node specific.

Reply:
ok so just one further thing I need to know: there are tests that use the DerEncode exports from util. Those tests are node-only. Is it ok to export the DerEncode methods from `ed25519` so the node-only tests can continue?

Reply:
Yes, it's fine to export for tests, just say so in the comments and use an underscore.

Reply:
Before this change this code would have run on any device from Earth to Endor. Therefore I dub this release `Salty Ewok` https://github.com/digitalbazaar/ed25519-verification-key-2018/pull/11/commits/90b1407efcf8154bfc7aa8f7297185c93ee27e86

May the emperor protect and bless you.